interchange/DeviceResources.capnp

```diff
@@ -627,6 +628,49 @@ struct Device {
     lutElements @1 : List(LutElements);
   }
 
+  ######################################
+  # BEL chains definitions
+  ######################################
+  struct BelChainsDefinitions {
+    struct BelChain {
+
+      struct ChainPattern {
+        source @0 : Text;
+	sink   @1 : Text;
+      }
+
+      struct ChainCoordConfig {
+        coord @0 : Text;
+	step  @1 : Int8;
+      }
+
+      struct ChainCellPort {
+        name  @0 : Text;
+	width @1 : Int8;
+      }
+
+      struct ChainCell {
+        cell        @0 : Text;
+	inputPins   @1 : List(ChainCellPort);
```

Comment:
it should be possible to get a list of ports from `primLibs` and/or the cell->bel mappings; I don't know why these are needed unless the intent is to refer to a subset of ports only?